substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/classinitialization/TypeInitializerGraph.java

```diff
@@ -216,7 +221,18 @@ private boolean isInvokeUnsafeIterative(InvokeTypeFlow i) {
     }
 
     private void addInitializer(AnalysisType t) {
-        types.put(t, initialTypeInitializerSafety(t));
+        ResolvedJavaType rt = t.getWrappedWithoutResolve();
```

Comment:
I would put this whole block in `initialTypeInitializerSafety`.

Reply:
fixed